platform-sdk/swirlds-platform-core/src/main/java/com/swirlds/platform/AddressBookInitializer.java

```diff
@@ -60,6 +62,8 @@ public class AddressBookInitializer {
     private static final String ADDRESS_BOOK_DIRECTORY_NAME = "address_book";
     /** The file name prefix to use when creating address book files. */
     private static final String ADDRESS_BOOK_FILE_PREFIX = "usedAddressBook";
+    /** The maximum number of address book files to keep in the address book directory. */
+    private static final int MAX_ADDRESS_BOOK_FILES = 50;
```

Comment:
We should make this into a setting (default 50 is ok though).

Reply:
made into a setting.